### ISSUE TITLE
Integration with npx?

### ISSUE THREAD

**Issue by fredericgermain** [mlwmlw/cheerio-cli#5]
This project is what I was looking for for cheerio !

Would it be possible to have this integrated cleanly with npx? It would need to have the cli to be the same name as the npm project... This would mean either push :
* either push the cli to cheerio project (my preference)
* or rename/add the cli cheerio-cli. this way we could just use `npx cheerio-cli ...`

Thanks a lot for writing this!

**Comment by mlwmlw** [comment]
I have published a new version 0.3.0 to support npx.  

```
$ curl -s https://www.google.com | npx cheerio-cli "title"
Google
```

**Comment by fredericgermain** [comment]
I never got one of my GitHub feature requests implemented so fast :D

Thank you so much, I confirm it's working now with cheerio-cli :+1:
